src/openrct2-win/openrct2-win.cpp

```diff
@@ -57,19 +56,9 @@ static char** GetCommandLineArgs(int argc, wchar_t** argvW)
             argv[i] = ConvertWideChartoUTF8(argvW[i]);
         }
     }
-    return argv;
-}
 
-static void FreeCommandLineArgs(int argc, char** argv)
-{
-    // Free argv
-    for (int i = 0; i < argc; i++)
-    {
-        free(argv[i]);
-    }
-    free(argv);
+    return argv;
 }
-
```

Comment:
Add this line back.

Reply:
I don't know why it's still failing

Reply:
It's still miscormatted. Either run `clang-format` locally or apply the patch from CI.

Reply:
It worked this time .Thanks for the tip